### ISSUE TITLE
[Feature Request] Changing a page's settings should be specific to the page by default

### ISSUE THREAD

**Issue by asdsadsaasd12312132** [darkreader/darkreader#4441]
Realistically, a user will only configure their global settings once or twice and leave it like that for basically forever. It seems far more appropriate for the settings to be specific to the page by default. Personally, I only change the settings to change the theme engine due to it to not working properly on a specific site.

**Comment by Gusted** [comment]
Hi @asdsadsaasd12312132!

I disagree with you on that, most users seems by-far to use the global engine and which edge cases to just use page settings to either change some settings make it look better. Also it isn't a hassle in the new UI to just switch to `site specific` settings what is 2 clicks on the main page.

Regards,
Gusted

**Comment by asdsadsaasd12312132** [comment]
Perhaps they use the global settings because they don't know about the site specific settings and are constantly being forced to change settings due their global settings working poorly with the page they are using. Think of this way, when playing video games, do people adjust their monitor brightness and contrast from their monitor or do they use the game-specific settings, assuming they exist? I would have to say that most people adjust the game settings and not their monitor settings. The same logic applies to Dark Reader. The global settings is the monitor and the site-specific settings are the websites.

**Comment by Gusted** [comment]
> Perhaps they use the global settings because they don't know about the site specific settings and are constantly being forced to change settings due to their global settings working poorly with the page they are using.

We are improving this on the new design 
![image](https://user-images.githubusercontent.com/25481501/101677244-35d8da00-3a54-11eb-9c9d-0742d8c12c74.png)
![image](https://user-images.githubusercontent.com/25481501/101677266-3ffad880-3a54-11eb-82e8-5029b8f50951.png)




> I just don't understand why it would be better to change the global settings over site specific settings once people have a configuration they are happy with. The global settings will look good on 90% of pages, so why change the global settings and potentially make most pages look worse instead of just changing the site-specific settings?

If people think the site doesn't look good they can simply enable the `Theme for site` option and change the settings whatever they like to.

> Another way of looking at it is when playing video games, do people adjust their monitor brightness and contrast from their monitor or do they use the game-specific settings, assuming they exist? I would have to say that most people adjust the game settings and not their monitor settings. The same logic applies to Dark Reader. The global settings is the monitor and the site-specific settings are the websites.

A issue with this comparison that those sites will have an consistent color-scheme generated by Dark reader based of the settings. Whereby sites have their own render engine and own understanding of the settings of the monitor and can interpreter it every way they feel to.


I still feel that when people changes their settings it's mostly for global usage rather than site-specific, however I don't have the data neither the number for this behaviour neither do we have tracking so we should have some user's commenting on this to understand what a default should be.

Regards,
Gusted

**Comment by nyuszika7h** [comment]
I agree, I very rarely want to mess with the global settings and only site-specific. Maybe have it remember what it was set to last time (global or per-site)?

**Comment by jc9108** [comment]
> so we should have some user's commenting on this to understand what a default should be

i also think site-specific should be default. i mean even the name implies it, global is what you set for all sites, it works for most sites, and then you make adjustments with site-specific settings for sites it doesnt work on. therefore, site-specific should be default logically. sure, now it's just 1 extra click, but just a qol thing.
